src/lib.rs

```diff
@@ -106,7 +111,8 @@ use libc::{c_int,pid_t};
 pub use ::os::target::{cc_t,speed_t,tcflag_t}; // types
 pub use ::os::target::{VEOF,VEOL,VERASE,VINTR,VKILL,VMIN,VQUIT,VSTART,VSTOP,VSUSP,VTIME}; // c_cc subscripts
 pub use ::os::target::{BRKINT,ICRNL,IGNBRK,IGNCR,IGNPAR,INLCR,INPCK,ISTRIP,IXANY,IXOFF,IXON,PARMRK}; // input modes
-pub use ::os::target::{OPOST,ONLCR,OCRNL,ONOCR,ONLRET,OFILL,NLDLY,NL0,NL1,CRDLY,CR0,CR1,CR2,CR3,TABDLY,TAB0,TAB1,TAB2,TAB3,BSDLY,BS0,BS1,VTDLY,VT0,VT1,FFDLY,FF0,FF1}; // output modes
+#[cfg(any(target_os = "linux", target_os = "macos"))] pub use ::os::target::{OPOST,ONLCR,OCRNL,ONOCR,ONLRET,OFILL,NLDLY,NL0,NL1,CRDLY,CR0,CR1,CR2,CR3,TABDLY,TAB0,TAB1,TAB2,TAB3,BSDLY,BS0,BS1,VTDLY,VT0,VT1,FFDLY,FF0,FF1}; // output modes
+#[cfg(target_os = "freebsd")] pub use ::os::target::{OPOST,ONLCR,TABDLY,TAB0,TAB3,ONOEOT,OCRNL,ONOCR,ONLRET}; // output modes
```

Comment:
The way the crate is organized, the constants exported here are supposed to be fairly cross platform. It was previously exporting just the POSIX-defined constants, but I didn't realize that BSD doesn't define some of them. I doubt any of the other constants are in common use, so I think it would be best to remove the other constants here for all OS's so that the interface remains the same on all platforms. The others will still be accessible through `termios::os`. At least then it's clear for a user when they're using a constant that won't compile on all platforms. This will be a breaking change, but I can bump the version to v0.2.